config/environments/production.rb

```diff
@@ -89,8 +89,7 @@
   config.log_formatter = ::Logger::Formatter.new
 
   # may be needed for integrating bootstrap with Heroku deployment
-  #config.cache_classes = true
-  #config.serve_static_assets = true
+  # config.cache_classes = true
```

Comment:
Yeah I actually didn't pay close attention to the comment about bootstrap :)
I'll get rid of that 😉